config/software/cmake.rb

```diff
@@ -1,16 +1,14 @@
 name "cmake"
 
-default_version "3.1.3"
+default_version "3.13.4"
 
-source :url => "http://www.cmake.org/files/v#{version[0..2]}/cmake-#{version}.tar.gz",
-       :md5 => "5697a77503bb5636f4b4057dcc02aa32",
-       :extract => :seven_zip
```

Comment:
any reason to remove `:extract => :seven_zip`? IIRC it's needed on windows to unzip `.tar.gz` files

Reply:
good catch, forgot to re-add after trying something, now it's back